src/styles/blocks/card.css

```diff
@@ -0,0 +1,84 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  background: var(--white-color);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  width: 200px;
+  height: 408px;
+  padding: 16px;
+}
+
+.card__image {
+  margin: 16px 3px 40px;
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  
+  margin: 0;
+  margin-bottom: 4px;
+}
+
+.card__code {
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+
+  color: var(--sec-text-color);
+  margin: 0;
+  margin-bottom: 16px;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.card__price-name {
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+
+  color: var(--sec-text-color);
+  margin: 0;
+}
+
+.card__price-amount {
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__button {
+  font-family: "Roboto", sans-serif;
```

Comment:
haven't you set it in html already?

Reply:
Yes I did, but if it's not set there it doesn't want to use it.